script/Arcueid/Day 6/06_00_ARC06_2.txt

```diff
@@ -312,17 +320,20 @@ And just like that, she makes eye contact with me, and then casually―――
 [c3b5da1eb76008b34752d572e17b7b9a77c62ce8]{
 -- Page 160, Offset 14312.
 -- 患部を見抜いているかと思うほど的確に、一番腫れている左足の甲を踏みやがったーーー！
-She steps on the most swollen part of my left foot with precision as if she could see through the affected area―――!
+She stomps right on the most swollen part of my left foot. Her aim is so accurate that you'd think she had a sixth sense for locating injuries―――!
```

Comment:
hmm, I see what you mean with foot vs. leg, maybe just blanket go with 'foot' even though I agree with my above comment about them being leg. Maybe you could coerce '踏み' into 'kick'

Reply:
I did actually go through and change everything to foot earlier, but then I got to the ministration scenes and his shin is what's most called out as beat to shit, and went back and converted them all back (except this one) and left the comment =/

I'm inclined to maybe leave this one as foot, since he does actually use a different Kanji for it (足 vs 脚).